org.sofproject.gst.topo/src/org/sofproject/gst/json/JsonCustomOptionPane.java

```diff
@@ -0,0 +1,156 @@
+package org.sofproject.gst.json;
+import java.awt.*;
```

Comment:
Do not use awt/swing. GUI should be based on swt as the other parts of Eclipse are.